packages/spor-react/src/theme/components/link.ts

```diff
@@ -14,7 +14,7 @@ const config = defineStyleConfig({
     backgroundPosition: "0 100%",
     backgroundRepeat: "no-repeat",
     borderRadius: "none",
-    pb: "2px",
+    p: "2px",
```

Comment:
We're trying to move away from the short hand notation. When we use both, there's not really anyway to know what padding will be applied. Please consider this small change:

```suggestion
    padding: "2px",
```